meinberlin/apps/plans/assets/PlansList.jsx

```diff
@@ -27,7 +27,7 @@ class PlansList extends React.Component {
   }
```

Comment:
I didn't change participationStatusClass to statusClass, i'm a bit confused if this is an artifact from somewhere? what is the best procedure in situations like this?

Reply:
Hmm, don't know. Change it back?